### ISSUE TITLE
QHexview displays the extension

### ISSUE THREAD

**Issue by neomissing** [Dax89/QHexView#53]
At present, Qhexview displays a number by default at 8bit. Is it possible to support 16bit display a number and 32bit display a number? May I ask which interface can be modified to achieve this，
![16bit](https://user-images.githubusercontent.com/22003930/106372434-93c9d400-63aa-11eb-9542-d221b3e9c1f6.png)
![32bit](https://user-images.githubusercontent.com/22003930/106372436-94626a80-63aa-11eb-9155-5b97d723bd40.png)

**Comment by audetto** [comment]
the key to display is

https://github.com/Dax89/QHexView/blob/master/document/qhexrenderer.cpp#L490

but, you really need to check everywhere you see a 3, you will have to replace with something, so that click, select and move work correctly.

**Comment by Dax89** [comment]
Rendering code must be changed in order to support various display modes, currently is hardcoded to 8 bit visualization.

I'm thinking to rewrite the entire rendering part because is old and very complex, it can be simplified a lot

**Comment by neomissing** [comment]
OK, thank you, I will look at the overall code, looking forward to the update of the new rendering QHexview

**Comment by Dax89** [comment]
Implemented in 5.0 branch: https://github.com/Dax89/QHexView/tree/5.0
This week I will merge it to `master`!

```cpp
/* From QHexDocument... */
document->setGroupLength(4); // It will be rendered as '00000000 00000000'

/* ...or from the widget itself */
hexview->setGroupLength(2);   // It will be renderer as '0000 0000 0000 0000'
```
